INFO     HTTP_CLIENT:event_hooks.py:30 Make POST request to http://localhost:8000/api/v1/users
INFO     HTTP_CLIENT:event_hooks.py:39 Got response 200 OK from http://localhost:8000/api/v1/users
INFO     HTTP_CLIENT:event_hooks.py:30 Make POST request to http://localhost:8000/api/v1/authentication/login
INFO     HTTP_CLIENT:event_hooks.py:39 Got response 200 OK from http://localhost:8000/api/v1/authentication/login
INFO     HTTP_CLIENT:event_hooks.py:30 Make GET request to http://localhost:8000/api/v1/files/incorrect-file-id
INFO     HTTP_CLIENT:event_hooks.py:39 Got response 422 Unprocessable Content from http://localhost:8000/api/v1/files/incorrect-file-id
INFO     BASE_ASSERTIONS:base.py:19 Check that response status code equals to 422
INFO     FILES_ASSERTIONS:files.py:135 Check get file with incorrect id response
INFO     ERROR_ASSERTIONS:errors.py:40 Check validation error response
INFO     BASE_ASSERTIONS:base.py:73 Check that length of "details" equals to 1
INFO     ERROR_ASSERTIONS:errors.py:19 Check validation error
INFO     BASE_ASSERTIONS:base.py:37 Check that "type" equals to uuid_parsing
INFO     BASE_ASSERTIONS:base.py:37 Check that "input" equals to incorrect-file-id
INFO     BASE_ASSERTIONS:base.py:37 Check that "context" equals to {'error': 'invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `i` at 1'}
INFO     BASE_ASSERTIONS:base.py:37 Check that "message" equals to Input should be a valid UUID, invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `i` at 1
INFO     BASE_ASSERTIONS:base.py:37 Check that "location" equals to ['path', 'file_id']
INFO     SCHEMA_ASSERTIONS:schema.py:22 Validate JSON schema